### PR TITLE
add rankings page

### LINKED ISSUE #59: Make rankings page header similar to other pages

### PR DESCRIPTION
**Pull request by vsvipul**
# What's this PR do? (related issue if exists)
closes #59 .. didn't add ratings yet because i didn't know how to calculate them. Will add them in subsequent PRs. 

# Screenshots? (necessary if an UI/UX change)
screenshot from 2019-02-08 19-14-58

### CODE REVIEW

File: controls/problems.js [hunk 1]
```diff
@@ -166,4 +167,50 @@ helper.postIde = function (req, res) {
     });
 }
 
+helper.renderUsers = function (req, res) {
```
Comment (Vishal1541) [inline controls/problems.js]: Add a few comments in this function as it looks a bit messy :sweat_smile: . Also, the table should display the users who have solved 0 question and the users with the same number of questions solved should have the same rank.

Comment (vsvipul) [reply controls/problems.js]: i have removed the code to fetch users and only the front end page is added in this PR.

Comment (vsvipul) [reply controls/problems.js]: you can work on the rendering part @Vishal1541 after merging this... This removes unnecessary files and makes the ranking page similar to other pages

Comment (Vishal1541) [reply controls/problems.js]: @vsvipul sure :+1:

**Pull request merged**